Tests/FunctionalCssTests/FunctionalCssTests.swift

```diff
@@ -5,14 +5,15 @@ import PointFreeTestSupport
 import SnapshotTesting
 import XCTest
 
+@MainActor
 class FunctionalCssTests: TestCase {
-  override func setUp() {
-    super.setUp()
-    //    SnapshotTesting.record = true
+  override func setUp() async throws {
+    try await super.setUp()
```

Comment:
Updated all tests to be `@MainActor` and `async throws`:

- XCTest basically requires things to be on the main thread, but async tests throw a wrench in this
- XCTest on Linux fails to build tests when async tests are mixed with non-async tests.